docs/content/modeling/organization-context-authorization.mdx

```diff
@@ -167,12 +174,13 @@ We will start with the following authorization model:
 <details>
 <summary>
 We are considering the case that:
-- Anne has a project manager role at organizations A, B and C
-- Beth has a project manager role at organization B
-- Carl has a project manager role at organization C
-- Project X is owned by organization A
-- Project X is shared with organization B
-
+<ul>
```

Comment:
Why this change?

Reply:
Just add a new line before the first bullet point, it should resolve it

Reply:
updated